kmk/boards/converter/keebio/viterbi_r2.py

```diff
@@ -13,3 +16,8 @@ class KMKKeyboard(_KMKKeyboard):
     split_type = 'UART'
     split_flip = True
     split_offsets = [7, 7, 7, 7, 7]
+
+    rgb_ext = RGB(pixel_pin=rgb_pixel_pin, num_pixels=12)
+    layers_ext = Layers()
+    split = Split(uart_pin=uart_pin, split_offsets=split_offsets)
+    extensions = [rgb_ext, split, layers_ext]
```

Comment:
After reading this through, I kinda get why this is done this way (as opposed to OOP). It's very readible and if you want anyone to write a new converter for a keeb, you dont want them to have to worry about inheritance patterns (although I think oop would be helpful on the overlapping extensions themselves)

Reply:
Decided that "boards" needed to come out anyways, and be a really slimmed down thing as most are just pinouts. Docs will give simple copy paste extentions on turning on RGB, or enabling splits ect.

Reply:
@hmoco fwiw a lot of the non-OOP patterns in this codebase are inherited from my general distaste for it. I stick to an OOP-lite style with FP influence in Python in general, and inheritance chains more than a layer or two deep tend to set off my spidey sense.

Reply:
@klardotsh fair enough! I totally dont feel like I have enough context that I can confidently say oop would be better haha. As far as spidey senses go, I can understand why several layers of inheritance would not be ideal, especially if these files are meant to be played with by people that aren't necessarily programmers (although the repetition triggered my spidey senses lol which is why I commented on it)

Reply:
I think that I've decided that these will be offered in a different way. I needed to get "boards" out of `/kmk` anyways as they were about to grow massive. I can offer many more iterations of choice in a git repo similar to how QMK does it, and still allow imports. This solves all of the "I don't have enough control" problems, as well as the "which microcontroller is which one" problem.